### PR TITLE
Utilize `tf_subnets`

### PR DESCRIPTION
**Pull request by SweetOps**
## What
* refactored to use `tf_subnets` module

## Why
* Better to reuse the `tf_subnets` module for maintenance

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -3,9 +3,9 @@ output "vpc_id" {
 }
 
 output "public_subnets" {
```
Comment (osterman) [inline outputs.tf]: I always go back and forth on this point. I struggle with calling it `_ids` (which is ugly, but clear) or without the suffix, which looks good, but maybe implies an object (of which is not a valid resource type to pass as an input). Should we rename it in `tf_subnets` to drop the `_ids`?

File: main.tf [hunk 2]
```diff
@@ -6,9 +6,20 @@ module "label" {
   stage     = "${var.stage}"
 }
 
+module "subnets" {
+  source = "git::https://github.com/cloudposse/tf_subnets.git?ref=master"
```
Comment (comeanother) [inline main.tf]: Use `tf_subnets` tag explicitly (e.g. current latest `tags/0.1.6`).

File: outputs.tf (hunk 1 above)
Comment (comeanother) [reply outputs.tf]: Rename `public_subnets` to `public_subnet_ids`

File: outputs.tf [hunk 3]
```diff
@@ -3,9 +3,9 @@ output "vpc_id" {
...
-  value = ["${aws_subnet.public.*.id}"]
+  value = ["${module.subnets.public_subnet_ids}"]
 }
 
 output "private_subnets" {
```
Comment (comeanother) [inline outputs.tf]: Rename `private_subnets` to `private_subnet_ids`

File: variables.tf [hunk 4]
```diff
@@ -19,3 +19,7 @@ variable "cidr" {
   description = "CIDR for the whole VPC"
```
Comment (comeanother) [inline variables.tf]: Rename `variable "cidr" {` to `variable "cidr_block" {`

Comment (SweetOps) [reply variables.tf]: has been renamed

**Pull request merged**